lib/yondu.rb

```diff
@@ -4,5 +4,28 @@
 
 module Yondu
   class Error < StandardError; end
-  # Your code goes here...
+
+  class MissingSetting < Error
+    def initialize(keys)
+      super("Missing setting: #{keys.join('->')}")
+    end
+  end
+
+  class Settings
+    def initialize(config)
+      @config = config
+    end
+
+    def get(*keys)
+      scope = []
```

Comment:
should we use a Set to avoid duplicates? or is not important?

Reply:
It could be possible.